src/Controller/EnMarche/TerritorialCouncil/OfficialReportManagerController.php

```diff
@@ -44,6 +44,13 @@ public function createAction(Request $request, OfficialReportManager $manager):
         ;
 
         if ($form->isSubmitted() && $form->isValid()) {
+            if (!$report->getAuthor()
```

Comment:
why do not move it on the top of the action to not allow to fill the form as it's not possible to submit it?

Reply:
We cannot, because we have no necessary information to make a check like political committee, that is accessible only after submitting a form, because its value can be chose by in the form.

Reply:
oh my bad 👍